examples/workshop/01-wind-solar.ipynb

```diff
@@ -0,0 +1,226 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Wind and Solar Hybrid Plant Example\n",
+    "---\n",
+    "In this example, we will walk through the process of simulating a hybrid renewable energy system using the Hybrid Optimization Performance Platform ([HOPP](https://github.com/NREL/HOPP)) library. We will simulate a hybrid system at a given location consisting of both wind and solar energy, and show how to access the simulation outputs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Required Modules\n",
+    "We start by importing the necessary modules and setting up our working environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/kbrunik/github/forked/HOPP/examples/workshop/log/hybrid_systems_2023-09-28T09.33.37.529683.log\n"
+     ]
+    }
+   ],
+   "source": [
+    "from hopp.simulation.hybrid_simulation import HybridSimulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Site Information\n",
+    "Set wind and solar resource data at plant location. In this example, we use the Flatirons site as an example location."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from hopp import ROOT_DIR\n",
+    "from hopp.simulation.technologies.sites import SiteInfo, flatirons_site\n",
+    "\n",
+    "DEFAULT_SOLAR_RESOURCE_FILE = ROOT_DIR.parent / \"resource_files\" / \"solar\" / \"35.2018863_-101.945027_psmv3_60_2012.csv\"\n",
+    "DEFAULT_WIND_RESOURCE_FILE = ROOT_DIR.parent / \"resource_files\" / \"wind\" / \"35.2018863_-101.945027_windtoolkit_2012_60min_80m_100m.srw\"\n",
+    "\n",
+    "\n",
+    "site = SiteInfo(\n",
+    "        flatirons_site,\n",
+    "        solar_resource_file=DEFAULT_SOLAR_RESOURCE_FILE,\n",
+    "        wind_resource_file=DEFAULT_WIND_RESOURCE_FILE,\n",
+    "        solar=True,\n",
+    "        wind=True,\n",
+    "        wave=False\n",
+    "    )"
+   ]
+  },
```

Comment:
That's a good point. My initial take is that we don't want these accessible via an official HOPP package install, as it would require us to package up data files, notebooks, and dependencies like Jupyter. However, once we publish a new version of HOPP, we can host these notebooks with something like Google Colab, which would let folks run and edit them as they explore the functionality. Open to suggestions here

Reply:
I think currently the whole examples folder is packaged up. I agree that they shouldn't be part of the package, so should be removed from `setup.py`. In which case the examples can only be run if you download the source code, and then the files should be available. Hosting these on Colab sounds great.